Apps/Admin/Client/Pages/SupportPage.razor.cs

```diff
@@ -100,7 +100,7 @@ private UserQueryType SelectedQueryType
                 return "Search parameter is required";
             }
 
-            if (this.SelectedQueryType == UserQueryType.Phn && !PhnValidator.Validate(StringManipulator.StripWhitespace(parameter)).IsValid)
+            if ((this.SelectedQueryType == UserQueryType.Phn || this.SelectedQueryType == UserQueryType.Dependent) && !PhnValidator.Validate(StringManipulator.StripWhitespace(parameter)).IsValid)
```

Comment:
Nit: this expression could be simplified by taking advantage of the PhnOrDependentSelected property added above.

Reply:
fixed